game/theater/conflicttheater.py

```diff
@@ -55,6 +58,8 @@ def __init__(self) -> None:
         self.ll_to_point_transformer = Transformer.from_crs(
             CRS("WGS84"), self.projection_parameters.to_crs()
         )
+        # Set default timezone as UTC
+        self.timezone = datetime.timezone(datetime.timedelta(hours=0))
```

Comment:
Better to not have a default so it's harder to forget to configure this with new maps. Could either make this an argument to the constructor or make an abstractmethod property the the descendants define, whichever you prefer.

Reply:
Roger, converted to a property as per the two adjacent examples. I note this isn't a strictly abstract property, but will at least throw a notimplementederror rather than silently creating rubbish kneeboards.